### DISCUSSION TITLE
User Op Backups / Revert

### DISCUSSION THREAD

**Discussion by andrejhronco** [cables-gl/cables_docs#166] [Feature Requests]
When developing an Op it would be great to be able to revert to a previously saved version.
Sometimes I break an op that I'm working on and then try and reload but i get into a state where the patch won't load because there's an error.

**Comment by ianww** [comment]
The same thing's happened to me!  Alternatively, or also, it would be good to have a way of opening up a patch in a non-processing mode, eg. by holding the esc key down, so that the problem op can be edited before processing is restarted.
